### PR TITLE
WPB-685 give SCIM connections human readable names

### PR DESCRIPTION
**Pull request by battermann**
## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

**Review by mdimjasevic — APPROVED** [review]
Looks good to me!

File: services/spar/src/Spar/Scim/Auth.hs [hunk 1]
```diff
@@ -122,9 +166,9 @@ createScimToken ::
   CreateScimToken ->
   Sem r CreateScimTokenResponse
 createScimToken zusr Api.CreateScimToken {..} = do
-  let descr = createScimTokenDescr
+  let descr = description
```
Comment (fisx) [inline services/spar/src/Spar/Scim/Auth.hs:169]: that could maybe be inlined..

File: libs/wire-api/src/Wire/API/Routes/Public/Spar.hs [hunk 2]
```diff
@@ -203,9 +216,10 @@ type APIScimTokenDelete =
...
 
+type APIScimTokenListV6 =
+  Get '[JSON] ScimTokenListV6
+
 data SparAPITag
 
 instance ServiceAPI SparAPITag v where
   type ServiceAPIRoutes SparAPITag = SparAPI
-  type SpecialisedAPIRoutes v SparAPITag = SparAPI
-  serviceSwagger = toOpenApi (Proxy @SparAPI)
```
Comment (fisx) [inline libs/wire-api/src/Wire/API/Routes/Public/Spar.hs:211]: where did this go?

**Review by fisx — APPROVED** [review]
LGTM.  Now we should do the same thing for saml idps, and then support multiple idps and multiple scim tokens in one team, and connecting them to each other.  there should be tickets for that, too, right?

File: cassandra-schema.cql [hunk 3]
```diff
@@ -2049,6 +2050,7 @@ CREATE TABLE spar_test.team_provisioning_by_token (
     descr text,
     id uuid,
     idp uuid,
+    name text,
```
[the remark this answers is not in the document]
Comment (battermann) [reply cassandra-schema.cql:2053]: shouldn't fix in this PR

**Pull request merged**